jcl/src/java.base/share/classes/openj9/tools/attach/diagnostics/base/DiagnosticUtils.java

```diff
@@ -302,7 +319,8 @@ private static DiagnosticProperties doHelp(String diagnosticCommand) {
 	@SuppressWarnings("nls")
 	private static final String DIAGNOSTICS_GC_CLASS_HISTOGRAM_HELP = "Obtain heap information about a Java process%n"
 			+ FORMAT_PREFIX + DIAGNOSTICS_GC_CLASS_HISTOGRAM + " [options]%n"
-			+ " Options: -all : include all objects, including dead objects%n"
+			+ " Options: all : include all objects, including dead objects (this is default option)%n"
```

Comment:
please put the `all` option on the next line.

Reply:
Moved `all` option to a new line.